pkl-core/src/test/files/LanguageSnippetTests/input/api/analyze1.pkl

```diff
@@ -0,0 +1,33 @@
+amends "../snippetTest.pkl"
+
+import "pkl:analyze"
+import "pkl:reflect"
+
+import ".../input-helper/analyze/a.pkl"
+import ".../input-helper/analyze/cyclicalA.pkl"
+import ".../input-helper/analyze/globImport.pkl"
+
+examples {
+  ["basic"] {
+    analyze.importGraph(Set(reflect.Module(a).uri))
+  }
+  ["cycles"] {
+    analyze.importGraph(Set(reflect.Module(cyclicalA).uri))
+  }
+  ["globs"] {
+    analyze.importGraph(Set(reflect.Module(globImport).uri))
```

Comment:
Considering that such examples are also often read as HOWTOs, I'd prefer just putting the paths in. This is a little "stilted," if you ask me.

Reply:
Not totally following this. What do you mean by paths?

Reply:
Addressed in other comment; I thought `analyze.importGraph("./other.pkl")` was intended legal, but it isn't.